pkg/quota/api/v1/types.go

```diff
@@ -25,7 +25,7 @@ type ClusterResourceQuotaSpec struct {
 	// and should only select active projects on the scale of dozens (though it can select
 	// many more less active projects).  These projects will contend on object creation through
 	// this resource.
-	Selector *unversioned.LabelSelector `json:"selector"`
+	Selector unversioned.LabelSelector `json:"selector"`
```

Comment:
It's not a map anymore: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/unversioned/types.go#L428-L435